src/components/vote-form.js

```diff
@@ -1,14 +1,121 @@
-import React from "react";
-import styled from "styled-components";
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-export default function VoteForm() {
-  return <Wrapper>안녕 나는 투표 폼!</Wrapper>;
+import styled from 'styled-components';
+
+function VoteForm() {
+	const [candi, setCandi] = useState([]);
+	const [voteCount, setVoteCount] = useState();
+
+	useEffect(() => {
+		getCandidates();
+	}, []);
+
+	const getCandidates = async () => {
+		await axios
+			.get(process.env.API_HOST + '/candidates/', candi)
+			.then(({ data }) => {
+				setCandi(data);
+			})
+			.catch(function (error) {
+				console.log(error);
+			});
+	};
+
+	const voteCandidates = async (person) => {
```

Comment:
person보다 candidate라는 parameter name이 좋을 것 같아요
```suggestion
	const voteCandidates = async (candidate) => {
```